### DISCUSSION TITLE
Unraid

### DISCUSSION THREAD

**Discussion by sohail579** [immich-app/immich#772] [Q&A]
Will there be a unraid image imminent?

**Comment by el-fredo** [comment]
I'll refer back to this slightly older post. Unfortunately, there is no template yet to easily install Unraid from the community app store.

But I followed this guide to get Immich running on Unraid: https://mfaz.dev/posts/immich-unraid/
It is a really simple step-by-step guide.

Since I have a "basic" CPU without AVX support, I had to adjust one line in the docker-compose.yml to get the machine-learning feature / object detection. See https://github.com/immich-app/immich/discussions/987 

As I said, Immich runs perfectly under Unraid, but two things could be improved:

1. A custom Docker network is created with the name "immich_default". Actually no problem at all, but it would be nice if Immich would use my already existing custom-network. Unfortunately, I don't know how to change this by adjusting files with the Docker Compose plugin. 
2. I still don't understand where the Immich data is stored when using the above installation method on Unraid. As an Unraid user you are used to the data being in `/mnt/user/AppData/` but that is not the case. Without knowing where the data is located, I don't know how to make a backup of my Immich installation.

I think we need more Immich enthusiasts who are familiar with Unraid and its Docker component. I have also tried to contact "Matt" who wrote the above step-by-step instructions for Unraid. Unfortunately, there is no contact listed on his website.
